README.md

```diff
@@ -90,6 +91,17 @@ Options can be combined:
 ./build.sh run --scale 7.0 --seed 42 'abstract art'
 ```
 
+On systems with <8GB of GPU RAM, you can try mixing and matching options:
+
+* Make images smaller than 512x512 using `--W` and `--H`
```

Comment:
Probably worth noting either here or in the options section, that the width and height need to be divisible by 64 (I think, this worked for me while simple multiples of 8 didn't).

Reply:
I added this to the option descriptions.